Supervisor/devserver.conf

```diff
@@ -1,8 +1,8 @@
 [program:devserver]
-command =  php -S external_ip:8888 -t /var/www/html/openchat/                 ; Command to start app
+command =  php -S 192.168.43.38:8888 -t /home/hiteshpr/Documents/kwoc/                 ; Command to start app
```

Comment:
Don't commit these files as you are modifying these files as per your requirement.